src/main/java/com/minecolonies/proxy/ClientProxy.java

```diff
@@ -67,6 +68,10 @@ public void showCitizenWindow(CitizenDataView citizen)
     @Override
     public void openBuildToolWindow(BlockPos pos)
     {
+        if (pos == null && Settings.instance.getActiveSchematic() == null) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)